### ISSUE TITLE
Variation summary - effect on protein

### ISSUE THREAD

**Issue by tuli** [WormBase/website#5604]
http://www.wormbase.org/species/c_elegans/variation/WBVar00241696#02-4563719-10-4

I will try to explain this as well as I can. 
Allele s71 is described in the paper as having a nonsense codon 281. 

Codon 281 is tta (Leucine). On the assumption that this is a single bp substitution, the molecular change will be either tta -> taa (Ochre) or tta -> tga (Opal). 

The curation is therefore ta -> rr (where r is the IUPAC code for 'a or g'

I think what the code is doing is working out all the possible outcomes ie
tta -> taa (1bp change - Nonsense Ochre)
tta -> tag (2bp change - Nonsense Amber)
tta -> tgg (2bp change - Missense Trp)
tta -> tga (1bp change - Nonsense Opal)

and the resulting Protein Effect is X (any amino acid). That protein effect is X is OK, but I don't think it should say Missense, because 'any amino acid' can also be a Stop (nonsense). 

Q. Is there a way the code can be changed to work out that this is a nonsense? If not then I suggest removing the Missense.

**Comment by sibyl229** [comment]
Hi @khowe , I'm not sure who can take a look at this, which I think is a slight data issue. The web code gets the "Missense" from the Molecular_change hash of the Predicted_CDS. It might also need a "Nonsense" label.

**Comment by khowe** [comment]
@sibyl229 you are right, this is a data/analysis issue. Not sure why is was labelled as a webteam issue. 

@Paul-Davis I'm not sure it's correct to represent the molecular change as ta->rr, because rr is 4 possible di-nucleotides, only 2 of which are observed.

**Comment by Paul-Davis** [comment]
@khowe I'm not 100% sure about the necessary change to fix this issue?

**Comment by Paul-Davis** [comment]
So it seems that the rr annotation was incorrect if the assumption is a single bp mutation,

```
genome agtTTAgcaga
Ochre     TAA
Opal      TGA
mut        *
annot      R
```

I have changed it for a single r in the WS264 build.

**Comment by Paul-Davis** [comment]
Does anything else need doing for this?
